nixnet/types.py

```diff
@@ -153,6 +154,15 @@ class LinComm(LinComm_):
     pass
 
 
+PduFrameReference_ = collections.namedtuple(
+    'PDU_',
+    ['ref', 'start_bit', 'update_bit'])
```

Comment:
Per my previous comment, this could probably use some renaming since we are moving away from references to objects. My suggestions:

PduFrameReference_ -> PduProperties_
PDU_ -> PduProperties_
ref -> pdu